### ISSUE TITLE
[Java-10-upgrade] Unable to start GoCD server with Java10 using the startup script

### ISSUE THREAD

**Issue by adityasood** [gocd/gocd#5166]
##### Issue Type
- Bug Report

##### Summary

We now support (v18.10.0 onwards) running GoCD server on Java10. However, the startup scripts are still not updated and error out while checking for installed jvmVersion.

###### Basic environment details

* Go Version: 18.10.0-7486
* JAVA Version: Java10
* OS: MacOs
* Browser vendor and version (if relevant): NA

##### Steps to Reproduce

1. upgrade your system java to java10
2. start the latest GoCD server
3. look for errors in go-server.out file

##### Expected Results
The server should start up normally

##### Actual Results
server startup fails

##### Possible Fix
change the validations for system jvm version to point to Java10

```~/Downloads/go-server-18.10.0  ./server.sh
[Wed Sep 12 15:06:00 IST 2018] Starting process:
[Wed Sep 12 15:06:00 IST 2018]   Working directory    : /Users/adityas/Downloads/go-server-18.10.0
[Wed Sep 12 15:06:00 IST 2018]   Application arguments: []
[Wed Sep 12 15:06:00 IST 2018]            GoCD Version: 18.10.0-7486
Running GoCD requires Java version 1.8. You are currently running with Java version 10. GoCD will now exit!```

**Comment by bdpiparva** [comment]
PR https://github.com/gocd/gocd/pull/5165 will fix the issue.

**Comment by ketan** [comment]
Closed via #5165
